src/languages.rs

```diff
@@ -1,151 +1,380 @@
-use strum::EnumIter;
-use strum::IntoEnumIterator;
+use std::str::FromStr;
+use strum::{Display, EnumIter, EnumString, IntoEnumIterator};
 
-#[derive(Debug, EnumIter)]
+#[derive(Debug, EnumIter, Display, EnumString)]
+#[strum(ascii_case_insensitive)]
 pub enum Iso639a {
```

Comment:
There are some removals in this enumerator, are they intentional, where they invalid languages?

Reply:
I went one at a time on this table and it is most likely that I missed one, or maybe not.
https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes